### PR TITLE
Star Wars BB9 

### PR DESCRIPTION
**Pull request by Kritin02**

### CODE REVIEW

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@Kritin02 Changes Required
1. Add snapshots of Solidworks files inside Mechanical Design folder.
2. Rename "Presentations and Reports" as "Poster and Report".
3. Add Project Report(in .pdf format) in "Poster and Report" folder.
4. There is some issue in opening your poster. Rectify it.
5. Rename files inside src. Names should be self-explanatory.
6. In README.md, add flowchart.
7. In README.md, Mechanical and Electronics aspect is not about components used. It should shortly explain about mechanical and electronics features of your project. You should add a separate sub-heading named "Components Used" to list out your components.
8. In README.md, hyperlink Team Members and Mentors name with their GitHub profile link.
9. In README.md, justify the text using HTML tag.

**Review by Kritin02 — COMMENTED** [review]
Ok

**Comment by sachin-ag** [comment]
All suggested changes have been made.

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@Kritin02 
1. Add "Models and Robotics Section" and its logo in Report.
2. Don't use images obtained from internet
3. Justify the text in README.md.You can do that using `<p align="justify">content<\p>` where content is the text you want to justify.

**Comment by sachin-ag** [comment]
Sir, I have made all the changes.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,146 @@
+<p align="justify">
```
Comment (agshubhs) [inline README.md]: @Kritin02 Remove this

File: README.md [hunk 2]
```diff
@@ -1,2 +1,146 @@
...
 # Star-Wars-BB9
 Srishti 2020
+
+<p align="center">
+  <img width="450" height="550" src="https://github.com/Kritin02/Star-Wars-BB9/blob/master/Images%20and%20Videos/Images/Interior.jpeg?raw=true">
+</p>
+
+## Abstract
+The droid should replicate BB-8 from the Star Wars franchise complete with a rotating body but a stationary head.The movement of the droid can be controlled by a companion app. The body can rotate freely and can be moved in all direction freely using the app.The head is established by using strong Neodymium magnets and ball casters.*
```
Comment (agshubhs) [inline README.md]: @Kritin02 
Remove asterisk sign from the end of the line and use p tag separately for each paragraph

File: README.md [hunk 3]
```diff
@@ -1,2 +1,146 @@
...
+* 1 X Arduino Uno
+* 3 X L298(Motor Driver)
+* 2 X 60rpm DC motors
+* 2 X 12V Li-polymer Batteries
+* Jumpers
+
+## Flowchart
+<img src="https://i.imgur.com/RjfrX5S.png">
+
+## Mechanical of the Design
```
Comment (agshubhs) [inline README.md:47]: Remove this images and add your solid works design these are copied images from internet.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,146 @@
...
+
+## Future Improvement
+* Could use camera on the head for survelliance.
+* Providing more sensors to prevent collision.
+* Use of PID, to improve the movement of the bot.
+* Could use advancedment in app to move the bot in desired shapes.
+* Use of GPS instead, to increase movement precision and to reach their destination.    
+
+## Team Members
+* <a href="https://github.com/Kritin02/">Kritin Agrawal</a>
```
Comment (agshubhs) [inline README.md]: Team members and mentors name should be in ordered list.

**Comment by agshubhs** [comment]
@Kritin02 mentioned me when you are done with the  changes

File: README.md (hunk 3 above)
Comment (sachin-ag) [reply README.md:47]: @Shubhanshu07 Sir this is not copied image I have made the flowchart and uploaded it into my Imgur account to use here

**Comment by sachin-ag** [comment]
@Shubhanshu07  Sir, I have made all the changes except regarding the images. All the images being used are ours. I have uploaded them to the file-hosting site Imgur so we can use them in the readme file. No one can access them without the link.

**Comment by agshubhs** [comment]
> @Shubhanshu07 Sir, I have made all the changes except regarding the images. All the images being used are ours. I have uploaded them to the file-hosting site Imgur so we can use them in the readme file. No one can access them without the link.

@sachin-ag @Kritin02  remove the form the site and add them here in github itself.

**Comment by sachin-ag** [comment]
@Shubhanshu07 Removed the copied images from readme file and added all required image in GitHub and linked them in the readme file as well.

**Comment by agshubhs** [comment]
@sachin-ag PR looks good to me. Great work guys

**Pull request merged**